build.gradle

```diff
@@ -65,4 +65,40 @@ dependencies {
 
 test {
 	useJUnitPlatform()
+}
+
+
+apply plugin: "java"
+
+repositories {
+	jcenter()
+}
+
+configurations {
+	ktlint
+}
+
+dependencies {
+	ktlint "com.github.shyiko:ktlint:0.27.0"
+	// additional 3rd party ruleset(s) can be specified here
```

Comment:
It's also nice that if we have any particular rulesets we want to add, gradle + ktlint should presumably make it easy to do so.